modules/siddhi-core/src/main/java/org/wso2/siddhi/core/query/processor/stream/function/StreamFunctionProcessor.java

```diff
@@ -42,6 +44,7 @@ protected void processEventChunk(ComplexEventChunk complexEventChunk, Processor
                     }
                     outputData = process(inputData);
                     complexEventPopulater.populateComplexEvent(complexEvent, outputData);
+                    break;
```

Comment:
no there's none. I didn't add it earlier but some examples i checked they
have put it. shall I remove it.. ?

On Tue, May 5, 2015 at 1:06 PM, Lasantha Fernando <notifications@github.com>
wrote:

> In
> modules/siddhi-core/src/main/java/org/wso2/siddhi/core/query/processor/stream/function/StreamFunctionProcessor.java
> <https://github.com/wso2/siddhi/pull/46#discussion_r29648359>:
>
> > @@ -42,6 +44,7 @@ protected void processEventChunk(ComplexEventChunk complexEventChunk, Processor
> >                      }
> >                      outputData = process(inputData);
> >                      complexEventPopulater.populateComplexEvent(complexEvent, outputData);
> > +                    break;
>
> Hi Damith,
>
> Usually we don't need to add a break for the default case that comes at
> bottom. Was there any specific reason for you to add a break for the
> default?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/wso2/siddhi/pull/46/files#r29648359>.
>



-- 
Software Engineer
WSO2 Inc.; http://wso2.com
<http://www.google.com/url?q=http%3A%2F%2Fwso2.com&sa=D&sntz=1&usg=AFQjCNEZvyc0uMD1HhBaEGCBxs6e9fBObg>
lean.enterprise.middleware

mobile: *+94728671315*